src/core/ERC721TL.sol

```diff
@@ -104,11 +106,6 @@ contract ERC721TL is
                                 Events
     //////////////////////////////////////////////////////////////////////////*/
 
-    /// @dev This event is for consecutive transfers per EIP-2309
```

Comment:
Aren't we keeping 2309 in the contract, just as an alternative option?

Reply:
Resorted to importing it from OpenZeppelin rather than manually defining.